tests/test_optimizely.py

```diff
@@ -544,6 +544,80 @@ def test_activate__with_attributes__audience_match(self):
     self._validate_event_object(mock_dispatch_event.call_args[0][0], 'https://logx.optimizely.com/v1/events',
                                 expected_params, 'POST', {'Content-Type': 'application/json'})
 
+  def test_activate__with_attributes_of_different_types(self):
+    """ Test that activate calls dispatch_event with right params and returns expected
+    variation when different types of attributes are provided and audience conditions are met. """
+
+    with mock.patch(
+        'optimizely.bucketer.Bucketer.bucket',
+        return_value=self.project_config.get_variation_from_id('test_experiment', '111129')) \
+        as mock_bucket, \
+      mock.patch('time.time', return_value=42), \
+      mock.patch('uuid.uuid4', return_value='a68cf1ad-0393-4e18-af87-efe8f01a7c9c'), \
+      mock.patch('optimizely.event_dispatcher.EventDispatcher.dispatch_event') as mock_dispatch_event:
+
+      attributes = {
+          'test_attribute': 'test_value_1',
+          'boolean_attribute': True,
```

Comment:
@mikeng13 This fails when I set it to False, because of the condition here: https://github.com/optimizely/python-sdk/blob/master/optimizely/event_builder.py#L186
This condition removes all attributes with falsy values including False, 0 and 0.0

Should I fix it in this PR?